demisto_sdk/commands/prepare_content/preparers/incident_to_alert.py

```diff
@@ -89,3 +90,31 @@ def edit_names_and_descriptions_for_playbook(
         new_content = re.sub(pattern, replace_with, new_content)
 
     return new_content
+
+
+def replace_playbook_access_fields_recursively(datum: Any) -> Any:
+
+    if isinstance(datum, list):
+        return [replace_playbook_access_fields_recursively(item) for item in datum]
+
+    elif isinstance(datum, dict):
+        for key, val in datum.items():
+            if isinstance(val, str):
+                if key in ["root", "simple"] and "incident" in val:
```

Comment:
does it handle `Incident`/`INCIDENT`? Do we want to handle those cases?

Reply:
No it doesn't. As far as I understand, we want to change only the `incident` access field, all other variations that might appear (e.g. input/output keys, etc.) shouldn't be changed. I plan to verify this again with Anar in the demo.